docs/modules/ROOT/examples/exoscale_iamkey.yaml

```diff
@@ -1,17 +1,17 @@
 apiVersion: exoscale.crossplane.io/v1
 kind: IAMKey
 metadata:
-  name: my-exoscale-iam-key
+  name: iam-key-local-dev
 spec:
   forProvider:
-    keyName: iam-key
+    keyName: iam-key-local-dev
     services:
       sos:
         buckets:
-          - bucket-test-1
+          - bucket-local-dev
     zone: CH-DK-2
   providerConfigRef:
     name: provider-config
   writeConnectionSecretToRef:
-    name: my-exoscale-iam-key
+    name: my-exoscale-user-credentials
     namespace: default
```

Comment:
This change was from `make generate`, not sure why this wasn't up to date. If you like I can move it to another PR.

Reply:
Someone probably forgot to run the generate command before merging to master. It's ok to be here...